CodeEditModules/Modules/Git/src/Client/Models/ChangedFiles.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ChangedFiles.swift
+//  
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import Foundation
+import SwiftUI
+import WorkspaceClient
+
+public struct ChangedFiles: Codable, Hashable, Identifiable {
+    public var id = UUID()
+    public var changeType: String
+    public var fileLink: String
```

Comment:
String are really dangerous generally, this should be an `URL` since it's a link